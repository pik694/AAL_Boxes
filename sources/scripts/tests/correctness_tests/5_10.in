5
99 37 39
82 2 71
41 61 26
72 91 13
100 96 7

