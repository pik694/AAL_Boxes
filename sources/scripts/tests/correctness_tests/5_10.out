1 4 2 3 0 
371903->360259
